templates/vue-demo-store/pages/checkout/index.vue

```diff
@@ -224,6 +224,18 @@ onMounted(async () => {
   });
 });
 
+const refreshAddresses = async (addressId?: string) => {
```

Comment:
There can be a situation where more than one address will be refreshed, for example, when you add a new address in a separate tab. I think it is better to add a general page loader, and remove `addressId` param from this function

Reply:
Improved this